flow/envs/base_env.py

```diff
@@ -505,46 +511,18 @@ def step(self, rl_actions):
             self.render()
 
         states = self.get_state()
-        if isinstance(states, dict):
```

Comment:
this is for multiagent right? why was this removed?

Reply:
oh i see, new multiagent base class. These changes should be in a separate PR imo